Docker Build Fails - Backend

**Describe the bug**
Backend build fails due to collate.go file.

**To Reproduce**
Steps to reproduce the behavior:
Launch docker-compose.yml

**Expected behavior**
`
#10 5.696 dbtools/collate.go:4:2: package backend/database_root is not in GOROOT (/usr/local/go/src/backend/database_root)
executor failed running [/bin/sh -c go build -o backend serverMain.go]: exit code: 1
ERROR: Service 'backend' failed to build : Build failed
docker-compose process finished with exit code 1
`

**Screenshot**
![Screenshot 2022-11-02 at 21 46 28](https://user-images.githubusercontent.com/69640923/199608796-a4e4119d-80a2-4ccd-883c-e5ca5bd3e88e.png)

--- a/backend/.dockerignore
+++ b/backend/.dockerignore
@@ -1,2 +1,1 @@
 serverMain
-database_root/**/*

--- a/backend/Dockerfile
+++ b/backend/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.18.0-alpine3.15 AS build
+FROM golang:1.18.2-alpine3.15 AS build
 WORKDIR /work
 COPY . .
 RUN \

--- a/frontend/Dockerfile
+++ b/frontend/Dockerfile
@@ -6,7 +6,7 @@ ENV API $API
 WORKDIR /usr/src/app
 
 # install and cache app dependencies
-COPY package.json package-lock.json .
+COPY package.json package-lock.json ./
 RUN npm install
 
 # copy the rest of the files
@@ -16,4 +16,4 @@ COPY . .
 RUN --mount=type=cache,target=/usr/src/app/.next/cache npm run build
 
 # start app
-CMD ["npm", "start"]
+CMD ["npm", "run", "dev"]

